tests/Aspire.Hosting.Tests/Aspire.Hosting.Tests.csproj

```diff
@@ -74,4 +74,8 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Aspire.Hosting.Redis.Tests" />
```

Comment:
Do we really need this? It is just tests - why not just make things `public` that we need to be public?

Reply:
Done as suggested